scripts/tasks.py

```diff
@@ -176,10 +176,10 @@ def find_peaks(config):
     pf.find_peaks()
     pf.curate_cxi()
     pf.summarize()
-    try:
-        pf.report(update_url)
-    except:
-        logger.debug("Could not communicate with the elog update url")
+    #try:
+    #    pf.report(update_url)
+    #except:
+    #    logger.debug("Could not communicate with the elog update url")
```

Comment:
Should we delete these lines to avoid future confusions as to why they are here?

Reply:
Good point. I'll get rid of old commented code.

Reply:
Thanks!